src/brogue/Items.c

```diff
@@ -6591,14 +6592,25 @@ static boolean useStaffOrWand(item *theItem) {
 
         rogue.featRecord[FEAT_PURE_WARRIOR] = false;
 
+        itemName(theItem, buf2, false, false, NULL);
+
         if (theItem->charges > 0) {
+
+            creature *monst = monsterAtLoc(zapTarget);
+            if (monst) {
+                monsterName(buf3, monst, true);
+                sprintf(buf, "you zap your %s at %s", buf2, buf3);
+            } else {
+                sprintf(buf, "you zap your %s", buf2);
```

Comment:
[required] These are missing full stops

```suggestion
                sprintf(buf, "you zap your %s at %s.", buf2, buf3);
            } else {
                sprintf(buf, "you zap your %s.", buf2);
```